src/app/prt-staking/provider.tsx

```diff
@@ -54,38 +58,69 @@ export const PrtStakingContextProvider = ({ children, token }: Props) => {
   const [cumulativeRevenue, setCumulativeRevenue] = useState<number | null>(
     null,
   )
-  const prtTokenAddress = token.prtTokenData.address as Address
-  const prtTokenDecimals = token.prtTokenData.decimals
+  const stakeTokenAddress = token.stakeTokenData.address as Address
+  const stakedTokenAddress = token.stakedTokenData.address as Address
```

Comment:
@janndriessen we'll likely need to tweak which one is used in the calls below during testing